intel_extension_for_transformers/llm/operator/cscr/dispatcher/CMakeLists.txt

```diff
@@ -1,26 +1,24 @@
-#  Copyright (c) 2023 Intel Corporation
-#
-#  Licensed under the Apache License, Version 2.0 (the "License");
-#  you may not use this file except in compliance with the License.
-#  You may obtain a copy of the License at
-#
-#    http://www.apache.org/licenses/LICENSE-2.0
-#
-#  Unless required by applicable law or agreed to in writing, software
-#  distributed under the License is distributed on an "AS IS" BASIS,
-#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-#  See the License for the specific language governing permissions and
-#  limitations under the License.
-
-
+##  Copyright (c) 2023 Intel Corporation
+##
+##  Licensed under the Apache License, Version 2.0 (the "License");
+##  you may not use this file except in compliance with the License.
+##  You may obtain a copy of the License at
+##
+##    http://www.apache.org/licenses/LICENSE-2.0
+##
+##  Unless required by applicable law or agreed to in writing, software
+##  distributed under the License is distributed on an "AS IS" BASIS,
+##  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+##  See the License for the specific language governing permissions and
+##  limitations under the License.
 cmake_minimum_required(VERSION 3.1 FATAL_ERROR)
 project(jblas_dispatcher LANGUAGES C CXX)
-set(CMAKE_CXX_FLAGS "-O3 -fPIC -Wno-narrowing")
+set(CMAKE_CXX_FLAGS "-O3 -fPIC -Wno-narrowing -fconcepts")
```

Comment:
will c++20 be too aggressive?

Reply:
concepts supported by lower-version gcc (e.g. gcc8.5), even if it's c++20 feature.

Reply:
according to my search, concept requires at least GCC10.
<html>
<body>
<!--StartFragment-->

Concepts | P0734R0 | 10 | __cpp_concepts >= 201907
-- | -- | -- | --


<!--EndFragment-->
</body>
</html>

Reply:
> according to my search, concept requires at least GCC10.
> 
> Concepts	P0734R0	10	__cpp_concepts >= 201907

https://gcc.gnu.org/onlinedocs/gcc-8.5.0/gcc.pdf
page 44

Reply:
I see. Then the flag is not a problem for me if it can work on GCC8.5.0